### DISCUSSION TITLE
PaXEvaluateIR returns zero

### DISCUSSION THREAD

**Discussion by lisabiermann** [FeynCalc/feyncalc#159] [Q&A] [answered]
Hi,

I cannot reproduce the result of `PaXEvaluateIR` that is shown in [1611.06793](https://arxiv.org/pdf/1611.06793.pdf) (p. 18) for `B0[SPD[p, p], 0, m^2]`. Executing the following just returns 0:
```
$LoadAddOns = {"FeynHelpers"};
<< FeynCalc`
PaXEvaluateIR[B0[SPD[p, p], 0, m^2], PaXSeries -> {{pp, m^2, 1}}, 
 PaXAnalytic -> True]
```
Could you potentially provide me with any insight on what might be going wrong on my side? (`PaXEvaluateUV` returns `1/epsUV` as expected.)
I'm using Mathematica 12.1, FeynCalc 9.3.1, FeynHelpers 1.3.0 on openSUSE Leap 15.3.

Thanks and cheers,
Lisa

**Comment by vsht** [comment, marked as answer]
Hi,

you are missing `SPD[p]=pp` as on p.17 of 1611.06793. Otherwise, `pp`  is not present inside the integral and there is nothing
to expand around.

Cheers,
Vlad

**Reply by lisabiermann** [reply]
Thanks a lot!
